go/tools/gazelle/rules/construct.go

```diff
@@ -77,7 +82,28 @@ func newValue(val interface{}) (bzl.Expr, error) {
 			list = append(list, elem)
 		}
 		return &bzl.ListExpr{List: list}, nil
-	default:
-		return nil, fmt.Errorf("not implemented %T", val)
+	case reflect.Struct:
+		if glob, ok := val.(globvalue); ok {
```

Comment:
following Go guard-clause style, this should be:
glob, ok := val.(globvalue)
if !ok {
  return nil, ... error ...
}